app/views/movies/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h3> Movie Information</h3>
+<h3> Title: <%= @movie.title %></h3>
+<h3> Year: <%= @movie.creation_year %></h3>
+<h3> Genre: <%= @movie.genre %></h3>
+<h5> Actors: <%= @movie.order_by_age %></h5>
+<h5> Actors Average Age: <%= @movie.avg_age %></h5>
+
+  <%= form_with url: "/movies/#{@movie.id}", method: :patch, data: {turbo: false} do |form| %>
```

Comment:
see note in update action around a more RESTful route. The path for this RESTful route could be `"/movies/:movie_id/actors"`